bin/pgenv

```diff
@@ -150,6 +151,9 @@ pgenv_check_dependencies(){
     pgenv_detail_command "sort" $(which sort)
     pgenv_detail_command "tr" $(which tr)
 
+    # check for git (required by build-git)
+    pgenv_detail_command 'git' $(which git)
```

Comment:
Is this checking for `git` for commands other than the new one? In general, I think that `pgenv check` should check everything, but individual commands should check only for the utilities they need Some folks will never want to build from the repo.

Reply:
Uhm...we need git anyway, because we suggest to update `pgenv` via `git`. Is therefore a problem to check for git? I'm really in doubt if this has to be removed or not...

Reply:
Heh, fair point. In general, though, I think we shouldn't require utilities unless the user actually needs to use them for particular commands.

Reply:
Ok, commit 3d9d12a moves the check out of the "standard" workflow and into the `build-dev` subcommand.